src/App.js

```diff
@@ -4,19 +4,22 @@ import './App.css'
 
 const App = () => {
   const [squares, setSquares] = useState(Array(9).fill(null))
-
+  const firstPerson = (index) => {
+    let updatedBoard = [...squares] 
+    updatedBoard[index]="🌴"
+    setSquares(updatedBoard)
+  }
   return (
     <>
       <h1>Tic Tac Toe</h1>
         <div className='gameboard'>
         {squares.map((value,index)=>{
 
-        return 
-        (
-          <Square value={value}key={index} />
+        return (
+        <Square firstPerson={firstPerson}value={value} key={index}
```

Comment:
We should circle back to fix this indentation before merging, good job otherwise.